### DISCUSSION TITLE
Rename build id to avoid long filename

### DISCUSSION THREAD

**Discussion by jakob30061** [nuxt/nuxt#25476] [Questions]
Hello,
i prerender my nuxt page so i get static files, which i serve from a small micro controller (esp8266).
Everything is working fine, till `_nuxt/builds/meta/11d1dc93-fc99-4b95-a106-40fb6b7ecd0a.json` is requested. My webserver cant serve this file, cause the path or filename is to long.

 If i edit the build files and rename the file with a shorter name everything is working fine.
 ```diff
-11d1dc93-fc99-4b95-a106-40fb6b7ecd0a.json
+test.json
```
 
 I couldn't find any information about what this file does, and for what nuxt needs that buildID.
 
 Is there a way to change that filename automatically or prevent `nuxt generate` to create it?
 I just want to look for simpler options than try to fix the webserver myself.
 
 maybe related to https://github.com/nuxt/nuxt/discussions/25448
 
 Thanks for your help in advance

**Comment by xjccc** [comment]
set this?
```
experimental: {
  appManifest: false
}
```

**Reply by TheAlexLichter** [reply]
This disables the feature but does not resolve the actual issue.

**Reply by xjccc** [reply]
if i set `ssr: false`, this is not necessary? it's right?

**Reply by jakob30061** [reply]
At first, thank you both for your help and the fast response time.

> set this?
> 
> ```
> experimental: {
>   appManifest: false
> }
> ```

This works for my edge case.
The /meta/${buildID}.json file is for caching am I right?
So that feature would be disabled by the experimental flag?

**Reply by xjccc** [reply]
emm..i find this https://github.com/nuxt/nuxt/pull/21641
